documentation/howtos.md

```diff
@@ -284,4 +284,33 @@ We also use the `juniper2` tag as a way to mark instances that have been updated
 
 The command will give you a summary of what it intends to do, and then prompt you for confirmation. It is recommended you add 
 the `--show-instances` argument to do a dry run and see the specific instances that will be upgraded. Any failures will be logged 
-and tagged for further examination.
\ No newline at end of file
+and tagged for further examination.
```

Comment:
@Kelketek, @nizarmah, this documentation change some how didn't get built and deployed on https://ocim.opencraft.com. :( I will check what happened and fix it.

Reply:
Did you have any luck finding the cause?

Reply:
@Kelketek, https://github.com/open-craft/opencraft/pull/687 if you want to review and approve :)

Reply:
@Kelketek @lgp171188 thanks for taking care of this 🙇🏼‍♂️